src/web/TextInput.tsx

```diff
@@ -122,16 +125,29 @@ export class TextInput extends React.Component<Types.TextInputProps, TextInputSt
                     onMouseUp={ this._checkSelectionChanged }
                     onPaste={ this._onPaste }
                     aria-label={ this.props.accessibilityLabel }
-                    type={ this.props.secureTextEntry ? 'password' : this._getKeyboardType() }
+                    type={ this.props.secureTextEntry ? 'password' : keyboardType }
                 />
             );
+
+            if (wrapInForm) {
+                // Wrap the input in a form tag if required
+                input = (
+                    <form action=''>
+                        { input }
+                    </form>
+                );
+            }
+
+            return input;
         }
     }
 
     private _getKeyboardType = () => {
```

Comment:
There's no longer a need for this to be a prebound delegate type. It can be a simple function.
```
private _getKeyboardType() {
```